### PR TITLE
Update react api

### PR DESCRIPTION
**Pull request by talyssonoc**

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -0,0 +1,11 @@
+{
+  "name": "reactjs/react-php-v8js",
+  "description": "PHP library that renders React components on the server",
+  "require": {
+    "ext-v8js": ">=0.1.3"
+  },
+  "license": "BSD",
```
Comment (zpao) [inline composer.json]: This should be "BSD-3-Clause"

**Comment by zpao** [comment]
Otherwise, :+1:

**Comment by talyssonoc** [comment]
Updated !

**Comment by zpao** [comment]
Thanks!

**Pull request merged**